proposals/lip-0041.md

```diff
@@ -539,26 +539,33 @@ The Auth module does not execute any logic during the genesis state finalization
 
 #### Transaction Verification
 
-As part of the verification of a transaction `trs`, the `nonce` and `signatures` properties are verified as follows.
+As part of the verification of a transaction `trs`, the `nonce` and `signatures` properties are verified as follows. During block processing, the transaction `trs` is only accepted if the return value of `verifyTransaction` equals `NONCE_VERIFY_STATUS_OK`. If `verifyTransaction` is called from the transaction pool, it is accepted if it is equal to `NONCE_VERIFY_STATUS_OK` or `NONCE_VERIFY_STATUS_PENDING`.
 
 ```python
-def verifyTransaction(trs: Transaction) -> None:
-    verifyNonce(trs)
+def verifyTransaction(trs: Transaction) -> int32:
```

Comment:
Is this how it is done in the SDK implementation?

In the block processing, you then need to ensure both that there is no exception thrown and also the return value is as expected.